examples/agentic/dummy_adaptive.py

```
@@ -0,0 +1,139 @@
+
+from pydantic import BaseModel
+
+logger = logging.getLogger(__name__)
+
+class PipelineContextDummy(BaseModel):
+    previous_score: float
+    current_score: float
+    generation: int
+
+class DummyProteinPipeline(ImpressBasePipeline):
+    def __init__(self, name: str, flow: Any, configs: Dict[str, Any] = {}, **kwargs):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)